### PR TITLE
Add katello 4.8 branching docs

### PR DESCRIPTION
**Pull request by chris1984**
Cherry-picked from @lfu
* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by ekohl** [comment]
Can you pick without `-x`? It wasn't merged to `master`, so it's not a real pick.

**Comment by chris1984** [comment]
@ekohl sorry about that, did a reset back to `3.6` and then did the `git cherry-pick xxxxxx` without the `-x`

File: guides/doc-Release_Notes/master.adoc [hunk 1]
```diff
@@ -14,7 +14,9 @@ endif::[]
 
 // Start inserting specific x.y.z releases here
 include::topics/foreman-3.6.1.adoc[leveloffset=+1]
-include::topics/foreman-3.6.0.adoc[leveloffset=+1]
```
Comment (ekohl) [inline guides/doc-Release_Notes/master.adoc:17]: You dropped 3.6.0 here, but that should stay in.

File: guides/doc-Release_Notes/topics/katello.adoc [hunk 2]
```diff
@@ -4,8 +4,19 @@
 [id="katello-headline-features"]
 == Headline Features
 
-[id="katello-upgrade-warnings"]
-== Upgrade Warnings
-
-[id="katello-deprecations"]
-== Deprecations
+* Inter Server Synchronization
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:7]: We use headings to refer to the parts so they  show up in the table of contents
```suggestion
=== Inter Server Synchronization
```

Then I'd also think about describing the feature in the heading. Something like
```suggestion
=== Syncable format for Inter Server Synchronization
```

File: guides/doc-Release_Notes/topics/katello.adoc [hunk 3]
```diff
@@ -4,8 +4,19 @@
...
+** Complete export has been supported for syncable format export. The hammer content-export option is only available for full content exports. But the content data managed by Katello can be measured easily over 100GB even TB which makes it time consuming and not efficient for complete exports every time.
+** Katello 4.8 adds the ability to perform incremental content exports in syncable format. All three content-export options are available in syncable format.
+  `hammer content-export incremental version --format=syncable`
+  `hammer content-export incremental library --format=syncable`
+  `hammer content-export incremental repository --format=syncable`
+** The export rpm files are generated as hardlinks to the actual file on disk to avoid the 2X space we may need for regular importable exports.
+* Content Views
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:14]: And here you could do something like
```suggestion
=== Register hosts to multiple Content Views
```

I'm unsure about the width in the TOC, but you can see in the preview what it looks like, or build it locally:
```sh
cd guides/doc-Release_Notes
make BUILD=katello html browser
```

File: guides/doc-Release_Notes/topics/katello.adoc [hunk 4]
```diff
@@ -4,8 +4,19 @@
 [id="katello-headline-features"]
 == Headline Features
 
-[id="katello-upgrade-warnings"]
-== Upgrade Warnings
-
-[id="katello-deprecations"]
-== Deprecations
+* Inter Server Synchronization
+** Complete export has been supported for syncable format export. The hammer content-export option is only available for full content exports. But the content data managed by Katello can be measured easily over 100GB even TB which makes it time consuming and not efficient for complete exports every time.
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello.adoc:8]: You can use regular paragraphs here, no need for a listing.

File: guides/doc-Release_Notes/master.adoc [hunk 5]
```diff
@@ -15,6 +15,9 @@ endif::[]
 // Start inserting specific x.y.z releases here
 include::topics/foreman-3.6.1.adoc[leveloffset=+1]
 include::topics/foreman-3.6.0.adoc[leveloffset=+1]
+ifdef::katello[]
```
Comment (ekohl) [inline guides/doc-Release_Notes/master.adoc:18]: We try to keep it chronological order (and perhaps Foreman needs to include dates too), so:
* F 3.6.1
* K 4.8.0
* F 3.6.0

**Comment by lfu** [comment]
@ekohl Thanks for reviewing. Updated.

File: guides/doc-Release_Notes/topics/katello-4.8.0.adoc [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+ * Last checkin and Registered columns should show up as empty if there's no data (https://projects.theforeman.org/issues/35854[#35854], https://github.com/Katello/katello/commit/3920e5a7384e93769dfa830b8097ed210c9a0e3d[3920e5a7])
+
+== Bug Fixes
+
+=== Inter Server Sync
+ * Importing incremental content not recreating metadata properly (https://projects.theforeman.org/issues/36164[#36164], https://github.com/Katello/katello/commit/df3b102e512c193b213ecc3a8c8ed2f2ef1ef8e6[df3b102e])
+ * Need a better warning message for empty incremental export (https://projects.theforeman.org/issues/36146[#36146], https://github.com/Katello/katello/commit/dde5565960ea6183f35a3aa10efe046b9636c5ae[dde55659])
+
+=== Repositories
+ * Need to update Recommended Repositories page with Satellite 6.13 repos (https://projects.theforeman.org/issues/36158[#36158], https://github.com/Katello/katello/commit/48694791d2922b52c39780529b812231090be9f5[48694791])
```
Comment (ekohl) [inline guides/doc-Release_Notes/topics/katello-4.8.0.adoc:28]: Not a blocker, but as a general guide I'd recommend removing mentions of Satellite and convert them to upstream friendly names. Something we should also keep reminding our Red Hat colleagues of.

**Review by ekohl — APPROVED** [review]
Thanks! I just kicked off the CI pipeline and will wait for that to complete so I can see the rendered document.

**Pull request merged**